### PR TITLE
Adding week 10 blog post

### PR DESCRIPTION
**Pull request by xtanion**
blog is visible in `blogs.html`

### CODE REVIEW

File: docs/source/posts/2022/2022-08-25-week-10-shivam.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Week 10 - Multi-node skinning support
+===================================
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-08-25-week-10-shivam.rst:2]: Shorter than the title, may give warning.

**Review by ganimtron-10 — COMMENTED** [review]
Nice Blog @xtanion , LGTM!

File: docs/source/posts/2022/2022-08-25-week-10-shivam.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+**In short: The InverseBindMatrix "undoes" any transformation that has already been applied to your model in its bind pose.** Note: Inverse bind matrix shouldn't be applied with weights.
+
+- I got the ``SimpleSkin`` model to work perfectly. Here's a preview:
+
+    .. raw:: html
+
+        <iframe id="player" type="text/html"   width="600" height="390" src="https://user-images.githubusercontent.com/74976752/186712241-eaf8d1b3-f983-4e21-8264-c191b3eb6e4b.mp4" frameborder="0"></iframe>
+
+- Support for multiple transformations is still missing, However ``RiggedSimple`` is also working fine, Except that It doesn't interpolate translation and scaling matrices.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-08-25-week-10-shivam.rst:25]: Should be `... still missing. However, ...`

Comment (m-agour) [reply docs/source/posts/2022/2022-08-25-week-10-shivam.rst:25]: Also `except that it doesn't`

File: docs/source/posts/2022/2022-08-25-week-10-shivam.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    .. raw:: html
+
+        <iframe id="player" type="text/html"   width="600" height="390" src="https://user-images.githubusercontent.com/74976752/186712256-af02c902-f60f-43b9-b257-f706c999557e.mp4" frameborder="0"></iframe>
+
+
+
+What is coming up next week?
+----------------------------
+
+- Adding tests and Adding support for combining transformation matrices for the same time stamp.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-08-25-week-10-shivam.rst:36]: Second `Adding` should be in lowercase

Comment (m-agour) [inline docs/source/posts/2022/2022-08-25-week-10-shivam.rst:36]: Should be `timestamp`

**Review by m-agour — COMMENTED** [review]
Thanks @xtanion for this informing blog post.
Please see my comments below.

**Review by skoudoro — COMMENTED** [review]
After you address @m-agour comments, it should be good to go. thanks @xtanion

**Comment by skoudoro** [comment]
Thank you for the update @xtanion. and thanks all for the review. merging

**Pull request merged**